### ISSUE TITLE
Cannot enable private BitBucket repository (Internal Server Error)

### ISSUE THREAD

**Issue by hranicka** [Shippable/support#1324]
When I click the "Enable" button for my repository (in the right bottom corner), I get `Internal Server Error`.

Caught requests/responses:

```
Remote Address:54.209.192.31:443
Request URL:https://api.shippable.com/workflow/enableRepoBuild
Request Method:OPTIONS
Status Code:200 OK

Response Headers
view source
Access-Control-Allow-Credentials:true
Access-Control-Allow-Headers:Accept,Content-Type,Authorization,Cookie
Access-Control-Allow-Methods:GET,PUT,POST,DELETE
Access-Control-Allow-Origin:*
Allow:POST
Connection:keep-alive
Content-Length:4
Content-Type:text/html; charset=utf-8
Date:Sat, 18 Apr 2015 07:37:22 GMT
X-Powered-By:Express

Request Headers
view source
Accept:*/*
Accept-Encoding:gzip, deflate, lzma, sdch
Accept-Language:cs-CZ,cs;q=0.8
Access-Control-Request-Headers:accept, authorization, content-type
Access-Control-Request-Method:POST
Connection:keep-alive
Host:api.shippable.com
Origin:https://app.shippable.com
Referer:https://app.shippable.com/
User-Agent:Mozilla/5.0 (Windows NT 6.1; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/41.0.2272.118 Safari/537.36 OPR/28.0.1750.51
```

And then:

```
Remote Address:54.209.192.31:443
Request URL:https://api.shippable.com/workflow/enableRepoBuild
Request Method:POST
Status Code:500 Internal Server Error

Response Headers
view source
Access-Control-Allow-Credentials:true
Access-Control-Allow-Headers:Accept,Content-Type,Authorization,Cookie
Access-Control-Allow-Methods:GET,PUT,POST,DELETE
Access-Control-Allow-Origin:*
Connection:keep-alive
Content-Length:21
Content-Type:text/html; charset=utf-8
Date:Sat, 18 Apr 2015 07:37:23 GMT
X-Powered-By:Express

Request Headers
view source
Accept:application/json, text/plain, */*
Accept-Encoding:gzip, deflate, lzma
Accept-Language:cs-CZ,cs;q=0.8
Authorization:apiToken <I've removed it in this log>
Connection:keep-alive
Content-Length:40
Content-Type:application/json;charset=UTF-8
Host:api.shippable.com
Origin:https://app.shippable.com
Referer:https://app.shippable.com/
User-Agent:Mozilla/5.0 (Windows NT 6.1; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/41.0.2272.118 Safari/537.36 OPR/28.0.1750.51

Request Payload
view source
{projectId: "552ac4935ab6cc1352c12a16"}
projectId: "552ac4935ab6cc1352c12a16"
```

**Comment by manishas** [comment]
@hranicka what permissions do you have to this repository? You need to be an owner to enable repos.

**Comment by hranicka** [comment]
Owner of the repo is "hranicka" account - mine. Not organization/group, just single account.

The same account that I use for Shippable login.

I use anothers 2 repos (hranicka/xxx) and they're fine. I've tried the third project now (hranicka/xxx) and everything was OK.

But when I try enable required project, I get an error (Internal Server Error) :(

(the problematic repo is a fork of the repository that I've enabled today without problems)
